pmg/views.py

```diff
@@ -210,6 +210,14 @@ def committee_detail(committee_id):
     """
     logger.debug("committee detail page called")
     committee = load_from_api('committee', committee_id)
+    YEAR_CUTOFF = 2012
```

Comment:
Let's sanity check this. We could also ask elasticsearch to give us stats... hmmm.